packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -130,6 +131,7 @@ export const Checkbox: FC<CheckboxProps> = ({
         <Pressable
           onPress={onPress}
           style={tile ? tileStyle : pressableBaseStyle}
+          accessibilityValue={accessibilityValue}
```

Comment:
Could be updated to `aria-valuetext` to align with newer/more web-like RN props.

Reply:
Nvm, couldn't here using the more general RN typing without updating that to string.